flake.nix

```diff
@@ -49,24 +53,26 @@
     pre-commit-hooks,
     poetry2nix,
     rust-overlay,
-  }: let
+    ...
+  } @ inputs: let
     inherit (nixpkgs) lib;
 
     SYSTEMS = [
-      flake-utils.lib.system.x86_64-linux
-      # TODO - Fix linux first and then get macos working.
-      # flake-utils.lib.system.x86_64-darwin
+      "x86_64-linux"
+      "x86_64-darwin"
+      "aarch64-linux"
+      "aarch64-darwin"
     ];
 
     kernelLib = import ./lib/kernels.nix {inherit self lib;};
   in
     (flake-utils.lib.eachSystem SYSTEMS (
       system: let
-        pkgs = nixpkgs.legacyPackages.${system};
+        pkgs = nixpkgs.legacyPackages.${system}.appendOverlays [
+          poetry2nix.overlay
```

Comment:
Since https://github.com/nix-community/poetry2nix/commit/51d10f73f1ae75e57f4f865548b79c439f981251 `poetry2nix` exports `overlays.default` instead of `overlay`.
So with the newer `poetry2nix`, this branch fails to build.

Reply:
thanks